sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContainer.md

```diff
@@ -0,0 +1,203 @@
+
+To retrieve Containers for the Microsoft Designer app, use OwningApplicationId: `5e2795e3-ce8c-4cfb-b302-35fe5cd01597`.
+
+
+
+### -----------------------EXAMPLE 2-----------------------------
+
+```powershell
+Get-SPOContainer -OwningApplicationId 423poi45-jikl-9bnm-b302-1234ghy56789 -Identity b66f5b2e-4cbd-4754-9ad3-8291c2c81ade 
+```
+
+Example 2 returns the detailed properties of the Container with associated `ContainerId`.
+
+ 
+### -----------------------EXAMPLE 3-----------------------------
+
+```powershell
+Get-SPOContainer -OwningApplicationId 423poi45-jikl-9bnm-b302-1234ghy56789 -Identity https://contoso.sharepoint.com/storageContainers/CSP_b66f5b2e-4cbd-4754-9ad3-8291c2c81ade 
+```
+
+Example 3 gives the detailed properties of a container using site URL of a container.
+
+
+### -----------------------EXAMPLE 4-----------------------------
+
+```powershell
+Get-SPOContainer -OwningApplicationId <OwningApplicationId> -Identity <ContainerId> -Paged | FT
+```
+
+Example 4 Uses the `-Paged` command to retrieve a paging token.
+
+### -----------------------EXAMPLE 5-----------------------------
+
+```powershell
+Get-SPOContainer -OwningApplicationId <OwningApplicationId> -Identity <ContainerId> -Paged -PagingToken <Token String> | FT 
+```
+
+Example 5 uses the `PagingToken` to view more Containers.
+
+## PARAMETERS
+
+### -OwningApplicationId
+
+This parameter specifies the ID of the SharePoint repository services application. Use `Get-SPOApplication` command to retrive the OwningApplicationId.
+ 
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+Applicable: SharePoint Online
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+
+### -Identity
+
+Use this parameter to specify the Container in the given OwningApplicationId.
+ 
+```yaml
+Type: String
+Parameter Sets: ParamSet4, ParamSet5
+Aliases:
+Applicable: SharePoint Online
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+
+### -Paged
+
+This parameter can be used when there are more than 5000 Containers in a given SharePoint repository services application. Using `-Paged` will provide a `<Paging Token>` that will display the next 5000 Containers.
+
+```yaml
+Type: String
+Parameter Sets: ParamSet2, ParamSet3
+Aliases:
+Applicable: SharePoint Online
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+
+### -PagingToken
+
+Use the this parameter to provide the `<Paging Token>` provided to view remaining Containers as shown in example 5. If there are no more Containers to display, the commandlet output will return the message `End of Containers view.` Otherwise, use the given `<Paging Token>` to retrieve the next batch of up to 5000 Containers.
```

Comment:
```suggestion
Use this parameter to provide the `<Paging Token>` provided to view the remaining containers as shown in Example 5. If there are no more containers to display, the commandlet output will return the message `End of Containers view.` Otherwise, use the given `<Paging Token>` to retrieve the next batch of up to 5,000 containers.
```